src/components/Table/TableHeader.jsx

```diff
@@ -5,9 +5,9 @@ import TableHeaderColumn from './TableHeaderColumn';
 
 const TableHeader = ({ columns, sortState, sortField, sortClick, totalWidth }) => (
   <MaterialTableRow>
-    {columns.map((column, index) => (
+    {columns.map(column => (
       <TableHeaderColumn
-        key={index}
+        key={column.field}
```

Comment:
I think column.field is empty string for some columns, there may be duplicate values.

Reply:
maybe column.field || column.name to have a backup